satpy/utils.py

```diff
@@ -574,29 +574,59 @@ def ignore_invalid_float_warnings():
         yield
 
 
-def get_chunk_size_limit(dtype):
-    """Compute the chunk size limit in bytes given *dtype*.
+def get_chunk_size_limit(dtype=float):
```

Comment:
Are you using `float` versus `np.float32` or `np.float64` for a specific reason?

Reply:
To keep it generic, but I could change if want to.